tests/manual-tests/lib/jquery/plugins/selectbox/VERSION.md

```diff
@@ -0,0 +1,5 @@
+The version of the jquery.selectbox plugin included in this folder was created from the master branch of the fluid-project fork:
+
+https://github.com/fluid-project/jquery.selectbox
```

Comment:
@cindyli what do you mean by original github repository? If you mean what we forked the code from, that place doesn't seem to exist anymore. We just happened to have the old version of the code in our codebase, so I moved that into a repo under fluid-project.